app/misc/graphing.ts

```diff
@@ -475,6 +475,7 @@ function makeAbsNode(c, column: number) {
     let email = stringer.split("%")[1];
     let flag = true;
     let count = 1;
+    let nodeId;
```

Comment:
I see this code is actually on the branch for issue 54, is this code from previous work that never got merged?

Reply:
@sterlinglaird yes, this fixed problem where tag and branch were missing in the second level zoom. It was fixed after our PR was merged..